packages/@react-stately/layout/src/TableLayout.ts

```diff
@@ -63,48 +65,87 @@ export class TableLayout<T> extends ListLayout<T> {
     ];
   }
 
-  buildColumnWidths() {
```

Comment:
The big change here is that we removed most all of the logic specifically for building column widths and put it into the `useColumnResizeWidthState` hook